docker/Dockerfile

```
@@ -1,12 +1,14 @@
 FROM oraclelinux:8
 
 LABEL author="OPERA ADT" \
-    description="s1 cslc 0.2 Gamma release" \
-    version="0.2-gamma"
+    description="s1 cslc 0.3 Gamma release" \
+    version="0.3-gamma"
```

```suggestion
    description="s1 cslc 0.3 calval release" \
    version="0.3-calval"
```
Or should it be delta?

---

The tag here is to clarify that this Docker image is delivered in the cal/val delivery, so I think I prefer to keep it as it is in this PR.